docs/Configuration.md

```diff
@@ -0,0 +1,43 @@
+# Configuring Arc.js
+The default configuration settings for Arc.js can be found in its `config/default.json` file.  Here is a description of the settings:
+
+**defaultVotingMachine**
+  The voting machine used by default by `Dao.new` when creating new DAOs.  Default is "AbsoluteVote".
+
+**gasLimit_deployment**
+Gas sent to transactions when migrating contracts and when creating DAOs that don't use a Unversal Controller.  A number like 4543760.
+
+**gasLimit_runtime**
+Gas sent to transactions in all cases except  those covere by gasLimit_deployment.  A number like 4543760.
+
+**network**
+Name of the blockchain network used during Arc contract migration.  Other information like url and port come from Arc.js's truffle.js file.  Default is "ganache".
+
+**providerUrl**
+The url to use when connecting to the blockchain network at runtime.  Default is http://127.0.0.1.
```

Comment:
you do have defaults here

Reply:
For the gaslimits I now reference the default.json file.